code/iomethods.py

```diff
@@ -518,5 +544,132 @@ def decode_time_from_string(timeString):
     print 'Error decoding time string: string does not match a predefined time format'
     return 0
     # **********************************************************************************************************************
+def write_np_array_to_ncdf(lon, lat, t, fileName, dirName):
+    '''
+        Purpose:: Convert a numPy array to netCDF
+
+        Inputs:: lon: a string representing the time units found in the file metadata
+                 lat: an integer representing the time interval found in the file's metadata
+                 t: A 3 dimensional numPy array holding temperatures in Kelvin.
+                    The first dimension is time and the second and third dimensions are longitude/latitude
+                 fileName: The name of the file to be written to
+                 dirName: Path to the directory to be written to
+        Returns:: None. It writes to a netCDF file with extension .nc
+    '''
+
+    ncdf = netCDF4.Dataset(dirName + '/' + fileName + '.nc', "w", format="NETCDF4")
+    ncdf.Conventions = "COARDS"  # Set global attributes
+    ncdf.calendar = "standard"
+    ncdf.comments = "File"
+    ncdf.model = "geos/das"
+    ncdf.center = "gsfc"
+
+    time = ncdf.createDimension("time", None)  # Create and set dimensions
+    longitude = ncdf.createDimension("longitude", size=9896)
+    latitude = ncdf.createDimension("latitude", size=3298)
+
+    longitudeVariable = ncdf.createVariable("longitude", "double", dimensions=("longitude",))  # Create and set variables
+    longitudeVariable.units = "degrees_east"                                                   # and their local attributes
+    longitudeVariable.long_name = "Longitude"
+
+    latitudeVariable = ncdf.createVariable("latitude", "double", dimensions=("latitude",))
+    latitudeVariable.units = "degrees_north"
+    latitudeVariable.long_name = "Latitude"
+
+    timeVariable = ncdf.createVariable("time", "double", dimensions=("time",))
+
+    ch4Variable = ncdf.createVariable("ch4", np.float, dimensions=("time","latitude","longitude",))
+    ch4Variable.comments = "Unknown1 variable comment"
```

Comment:
Edit, here to passing the list of variables. Maybe a list of dictionaries, where the dict contains all of this info for a variable.